meinberlin/assets/scss/components/_hero_unit.scss

```diff
@@ -0,0 +1,40 @@
+.hero-unit {
+    position: relative;
+    min-height: 25em;
+
+    background: no-repeat center center;
```

Comment:
This implicitly overwrites all other background properties. I would prefer `background-size`/`background-position`